### PR TITLE
fix: update environment variable files in Docker Compose configurations

### PR DESCRIPTION
**Pull request by saalim-mushtaq**

### CODE REVIEW

**Comment by saalim-mushtaq** [comment]
@adnan-kamili

File: .env [hunk 1]
```diff
@@ -7,22 +7,21 @@ POSTGRES_DB=cryptlex
 # LetsEncrypt SSL certificate
 
 EMAIL=joe.doe@mycompany.com
-WEB_API_DOMAIN=cryptlex-api.mycompany.com
-DASHBOARD_DOMAIN=cryptlex-app.mycompany.com
-RELEASE_SERVER_DOMAIN=cryptlex-releases.mycompany.com
+WEB_API_DOMAIN=api-test.cryptlex.io
```
Comment (adnan-kamili) [inline .env]: why is the cryptlex.io domain being used here?

File: .env [hunk 2]
```diff
@@ -7,22 +7,21 @@ POSTGRES_DB=cryptlex
...
+RESELLER_PORTAL_DOMAIN=reseller-test.cryptlex.io
 
 # filestore - Minio or AWS S3
 
 FILE_STORE_ACCESS_KEY=admin
 FILE_STORE_SECRET_KEY=1password
 
-# Google SSO
-
-GOOGLE_CLIENT_ID=PASTE_GOOGLE_CLIENT_ID
```
Comment (adnan-kamili) [inline .env]: why is this removed

File: web-api.env [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Wep API server settings
 PORT=5000
 FORCEHTTPS=false
-RSA_PASSPHRASE=PASTE_RSA_SECRET
-APPLICATION_LICENSE_KEY=PASTE_LICENSE_KEY
+RSA_PASSPHRASE=aAbBcCdDeEfF
+APPLICATION_LICENSE_KEY=A2CE5C-F4A70D-47FF85-261A3B-ACF503-8685C6
```
Comment (adnan-kamili) [inline web-api.env]: why has this file been changed? this leakes the license key and secret in a public repo

File: docker-compose.yml [hunk 4]
```diff
@@ -150,7 +150,79 @@ services:
...
+      - backend
+    restart: unless-stopped
+    ports:
+      - 4400
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "20m"
+    labels:
+      - traefik.backend=cryptlex/cryptlex-admin-portal
```
Comment (adnan-kamili) [inline docker-compose.yml]: please clarify why backend uses this value? Have we used this pattern anywhere in the file

File: docker-compose.yml [hunk 5]
```diff
@@ -150,7 +150,79 @@ services:
       - traefik.docker.network=cryptlex-on-premise_backend
       - traefik.port=4200
 
-  release-server:
+ admin-portal:
```
Comment (adnan-kamili) [inline docker-compose.yml]: the indentation is incorrect which makes the file invalid. YAML is sensitive to indentation

**Review by adnan-kamili — CHANGES_REQUESTED** [review]
please check comments

File: docker-compose.yml [hunk 6]
```diff
@@ -150,6 +150,78 @@ services:
...
+
+  reseller-portal:
+    image: cryptlex/reseller-portal:latest
+    depends_on:
+      - web-api
+    env_file: web-portals.env
+    environment:
+      WEB_API_BASE_URL: https://${WEB_API_DOMAIN}
+      RELEASE_SERVER_BASE_URL: https://${RELEASE_SERVER_DOMAIN}
+
```
Comment (adnan-kamili) [inline docker-compose.yml]: remove this extra line

File: docker-compose.yml [hunk 7]
```diff
@@ -150,6 +150,78 @@ services:
...
 
+  admin-portal:
+    image: cryptlex/admin-portal:latest
+    depends_on:
+      - web-api
+    env_file: web-portals.env
+    environment:
+      WEB_API_BASE_URL: https://${WEB_API_DOMAIN}
+      RELEASE_SERVER_BASE_URL: https://${RELEASE_SERVER_DOMAIN}
+
```
Comment (adnan-kamili) [inline docker-compose.yml]: remove this extra line

File: docker-compose.yml [hunk 8]
```diff
@@ -150,6 +150,78 @@ services:
...
+  customer-portal:
+    image: cryptlex/customer-portal:latest
+    depends_on:
+      - web-api
+    env_file: web-portals.env
+    environment:
+      WEB_API_BASE_URL: https://${WEB_API_DOMAIN}
+      RELEASE_SERVER_BASE_URL: https://${RELEASE_SERVER_DOMAIN}
+
```
Comment (adnan-kamili) [inline docker-compose.yml]: remove this extra line

**Review by adnan-kamili — CHANGES_REQUESTED** [review]
please check comments

**Pull request closed without merging**